composer.json

```diff
@@ -30,16 +30,18 @@
 		}
 	},
 	"scripts": {
-		"cs-check": "phpcs --colors --parallel=16 -p src/",
-		"cs-fix": "phpcbf --colors --parallel=16 -p src/",
-		"phpstan": "phpstan.phar analyse",
-		"psalm": "psalm.phar --show-info=false",
+		"cs-check": "phpcs --colors --parallel=16 -p src/ tests/",
+		"cs-fix": "phpcbf --colors --parallel=16 -p src/ tests/",
+		"phpstan": "tools/phpstan analyse",
+		"psalm": "tools/psalm --show-info=false",
 		"stan": [
 			"@phpstan",
 			"@psalm"
 		],
+		"stan-baseline": "tools/phpstan --generate-baseline",
+		"psalm-baseline": "tools/psalm --set-baseline=psalm-baseline.xml",
 		"test": "phpunit",
-		"stan-setup": "cp composer.json composer.backup && composer require --dev phpstan/phpstan:~1.9.0 psalm/phar:~5.4.0 && mv composer.backup composer.json",
+		"stan-setup": "phive install",
```

Comment:
I've introduced the phive toolset which we also use in the core.
See https://github.com/cakephp/cakephp/blob/5.x/.github/CONTRIBUTING.md#test-cases-codesniffer-and-static-analysis to learn more about it